### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by amire80**

### CODE REVIEW

**Comment by lukeapage** [comment]
Thanks for all the changes, though note we are working on replacing the current site with the one at less-docs. I'll keep the relevant ones open till I can apply to both repos

**Comment by lukeapage** [comment]
https://github.com/less/less-docs

**Comment by amire80** [comment]
Hmm. This is quite confusing, because the description of less-docs says "This will be moved to the lesscss.org repo soon!"

**Comment by seven-phases-max** [comment]
> because the description of less-docs says "This will be moved to the lesscss.org repo soon!"

Which means that all the contents of lesscss.org will be overwritten by less/less-docs

@lukeapage maybe we could put some notice into https://github.com/less/lesscss.org/blob/master/README marking it as obsolete to prevent people from wasting their time fixing it?

**Comment by jonschlinkert** [comment]
>  maybe we could put some notice into https://github.com/less/lesscss.org/blob/master/README to mark it as obsolete to prevent people of wasting their time fixing it?

:+1: good idea

**Comment by jonschlinkert** [comment]
and thanks a lot for doing this @amire80! I hope you continue contributing

File: templates/pages/changes.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ We have released 1.4.0. This includes new features such as extends, the data-uri
 
 There are some known <span class="warning">breaking changes</span>.
 
-`@import-once` is removed and is now default behaviour for `@import`.
+`@import-once` is removed and is now default behavior for `@import`.
```
Comment (lukeapage) [inline templates/pages/changes.md:97]: I think I disagree - most of less uses american english I think.

http://grammarist.com/spelling/behavior-behaviour/

In applying to less-docs I have actually changed the couple of places using behaviour to behavior

**Pull request closed without merging**